docs/operator-manual/applicationset/Generators-Pull-Request.md

```diff
@@ -178,6 +178,12 @@ spec:
           passwordRef:
             secretName: mypassword
             key: password
+        # Credentials for Bearer Token (App Token) authentication. Either basicAuth or bearerToken
+        # authentication is required to access private repositories
+        bearerToken:
+          tokenRef:
+            secretName: repotoken
+            key: token
```

Comment:
Can you add descriptions of the this fields below just like we have for Basic Auth?

Reply:
It should be clear by the description above in line 181/182, see for example 137 where it is similar right? We shouldn't have to explain a secretRef right?

Reply:
BTW, the description is copied from 234/235 to keep it aligned

Reply:
I see the source of your reference. However, even regarding line 137 that you mentioned, line 151 contains a concise explanation for `tokenRef`.

I'm requesting to include something similar to the example found at https://github.com/argoproj/argo-cd/blob/4fdeb1efb74d77fa65f5bf620500c5e4e7d98715/docs/operator-manual/applicationset/Generators-Pull-Request.md?plain=1#L261-L262.

Reply:
Agreed, overlooked that part. I've updatet, the PR.